web-next/app/globals.css

```
@@ -0,0 +1,960 @@
+  align-items: center;
+  gap: 10px;
+  margin-bottom: 14px;
+}
+.hero-eyebrow::before {
+  content: "";
+  width: 32px;
+  height: 1px;
+  background: var(--acc);
+}
+.hero h1 {
+  font-family: var(--font-display), "Syne", sans-serif;
```

Stylelint (17.6.0)

[error] Unexpected quotes around "Syne" (font-family-name-quotes)

(font-family-name-quotes)